app/controllers/decidim/votings/vote_confirmation/confirmations_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    module VoteConfirmation
+      class ConfirmationsController < Decidim::Votings::VoteConfirmation::ApplicationController
+        def confirm
+          voter_id = params[:uid]
+          render json: { result: "ko" } and return unless voter_id
```

Comment:
I don't think that nVotes server is expecting any response

Reply:
It's only for debugging purposes

Reply:
:+1: